src/Components/Attributes/Attributes.js

```diff
@@ -105,7 +109,17 @@ class Attributes extends React.Component {
     // `dataStructures` prop is optional and is used to
     // resolve inheritance, references and includes/mixins, plus to
     // render the inheritance tree.
-    const dataStructures = props.dataStructures || [];
+    let dataStructures = props.dataStructures || [];
+    let element = props.element;
+
+    if (isFunction(element.getMetaProperty)) {
+      // serialize minim Element inputs (`element` and `dataStructures`)
+      // to the expected refract 0.6 serialization
+      const refractSerializer = new JSON06Serialiser(minim.namespace());
+      element = refractSerializer.serialise(element).content[0];
+      dataStructures = dataStructures.map(d => refractSerializer.serialise(d).content[0]);
```

Comment:
Perhaps the map could be simplified for readability (and order of operation to be separate). Something like:

```js
dataStructures
  .map(dataStructure => dataStructure.get(0))  // Selects the first element inside the dataStructure
  .map(refractSerializer.serialise);
```

Reply:
Note this comment is invalid after my final review comment.